src/main/flight/pid.c

```diff
@@ -609,8 +624,23 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
         }
 
         // -----calculate I component
+        float itermErrorRate;
+        if (itermRelax) {
+            const float gyroTargetHigh = pt1FilterApply(&windupLpf[axis][0], currentPidSetpoint);
+            const float gyroTargetLow =  pt1FilterApply(&windupLpf[axis][1], currentPidSetpoint);
+            DEBUG_SET(DEBUG_ITERM_RELAX, 0, gyroTargetHigh);
+            DEBUG_SET(DEBUG_ITERM_RELAX, 1, gyroTargetLow);
+            const float gmax = MAX(gyroTargetHigh, gyroTargetLow);
+            const float gmin = MIN(gyroTargetHigh, gyroTargetLow);
+            if (gyroRate >= gmin && gyroRate <= gmax)
```

Comment:
Please always use blocks after conditional / loop statements. This makes maintenance of the code less prone to bugs.